src/lib.rs

```diff
@@ -150,9 +166,27 @@ impl ApplicationBuilder {
         self.log_settings.everything = b; self
     }
 
+    pub fn runcommand (&mut self, command: Option<std::path::PathBuf>) -> &mut Self {
```

Comment:
Even though this is fine as it is, I was thinking of having `ApplicationBuilder::start` have a optional argument to define the path instead. Whats your thoughts on it?

Reply:
It depends on how we'll handle command arguments.

Reply:
Besides the argument parser has some annoying behaviour:

    cmd -a 1 -b 2 non-arg non-arg -c 3 -- non-arg

For some reason the parsing continues after first non option argument "non-arg". Also a couldn't find a way to get all non option arguments: in this case "non-arg"x3.